src/card.css

```diff
@@ -0,0 +1,73 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3e5;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__item {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin: auto;
+  color: #060b35;
+}
+
+.card__code {
+  font-size: 10px;
+  font-weight: 400;
```

Comment:
it's default value

Reply:
@ivanchuk-yurii thank you for your advise.